src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -221,45 +221,13 @@ Note that electrode positions SHOULD NOT be added to this file but to `*_electro
 
 The columns of the channels description table stored in `*_channels.tsv` are:
 
-MUST be present **in this specific order**:
-
 <!-- This block generates a columns table.
 The definitions of these fields can be found in
-  src/schema/objects/columns.yaml
+  src/schema/rules/tabular_data/*.yaml
 and a guide for using macros can be found at
  https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
 -->
-{{ MACROS___make_columns_table(
-   {
-      "name__channels": ("REQUIRED", "When a corresponding electrode is specified in `_electrodes.tsv`, "
-                         "the name of that electrode MAY be specified here and the reference electrode "
-                         "name MAY be provided in the `reference` column."),
-      "type__ieeg_channels": "REQUIRED",
-      "units": "REQUIRED",
-      "low_cutoff": "REQUIRED",
-      "high_cutoff": "REQUIRED",
-   }
-) }}
-
-SHOULD be present:
-
-<!-- This block generates a columns table.
-The definitions of these fields can be found in
-  src/schema/objects/columns.yaml
-and a guide for using macros can be found at
- https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
--->
-{{ MACROS___make_columns_table(
-   {
-      "reference__ieeg": "OPTIONAL",
-      "group__channel": ("OPTIONAL", "Note that any groups specified in `_electrodes.tsv` must match those present here."),
-      "sampling_frequency": "OPTIONAL",
-      "description": "OPTIONAL",
-      "notch": "OPTIONAL",
-      "status": "OPTIONAL",
-      "status_description": "OPTIONAL",
-   }
-) }}
+{{ MACROS___make_columns_table("ieeg.iEEGChannels") }}
```

Comment:
Same issue as eeg type render above, the following are present in current spec, but not this render:
```
Must be one of: "MEGMAG", "MEGGRADAXIAL", "MEGGRADPLANAR", "MEGREFMAG", "MEGREFGRADAXIAL", "MEGREFGRADPLANAR", "MEGOTHER", "EEG", "ECOG", "SEEG", "DBS", "VEOG", "HEOG", "EOG", "ECG", "EMG", "TRIG", "AUDIO", "PD", "EYEGAZE", "PUPIL", "MISC", "SYSCLOCK", "ADC", "DAC", "HLU", "FITERR", "OTHER".
```

Reply:
See associated glossary entry @bendhouseart